app/customerstatusmanager.py

```diff
@@ -8,18 +8,19 @@ class CustomerStatusManager():
     '''
 
     def change_status(self, customer):
-        with sqlite3.connect('../bangazon.db') as conn:
+        with sqlite3.connect('bangazon.db') as conn:
             c = conn.cursor()
 
             try:
                 c.execute("UPDATE Customers SET active=0 WHERE active=1")
-                c.execute("UPDATE Customers SET active=1 WHERE customer_name = '{}'".format(customer.get_customer_name()))
+                c.execute("UPDATE Customers SET active=1 WHERE customerId = '{}'".format(customer))
                 c.execute("SELECT active FROM Customers WHERE active=1")
                 status = c.fetchall()[0][0]
+                print(status)
```

Comment:
May want to eventually get rid of this print statement since it was only used for debugging.